File: foundation 2 month/p_exam/10-10-24/v3/ex_2.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>
#include <time.h>
int enter()
{
	return rand()%100;
}
void check(int ms[],int k)
{
	int sum=0;
	for(int i=0;i<k;i++)
	{
		sum+=ms[i];
	}
	printf("Massivlar yigindisi: %d",sum);
}
int main()
{
	srand(time(0));
	int n;
	printf("Elelemtnlar soni: ");
	scanf("%d",&n);
	int arr[n];
	for(int i=0;i<n;i++)
	{
		arr[i]=enter();
		printf("%4d",arr[i]);
	}
	puts("");
	check(arr,n);
}

File: full stack/full stack 1 month/lesson3/classwork/ex_2.js
let a = 15
let b = 20
let sum = a+b
if ( sum > 0 && sum <24)
{
    console.log(sum)
}
else 
{
    if (a>b)
    {
        console.log(24-(a+b))
    }
    else
        b=24-(a+b)
        b=b*(-1)
        console.log(b)
}